### PR TITLE
add post-robot@4.0.34 w/ git auto-update

### LINKED ISSUE #9977: [Request] Add post-robot
post-robot
https://github.com/krakenjs/post-robot
https://www.npmjs.com/package/post-robot

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/40096776-request-add-post-robot?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by x09326**
close #9977, cc @bluepnume
@kennynaoh please help me review it, thanks.

Use git auto-update because github release more versions than npm.

Pull request for issue: #9977 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/krakenjs/post-robot
 * Official website (optional, not the repository):
 * NPM package url (optional):https://www.npmjs.com/package/post-robot
 * License and its reference:Apache-2.0
Ref: https://github.com/krakenjs/post-robot/blob/master/LICENSE.txt
 * GitHub / Bitbucket popularity (required):
   - Count of stars:50
   - Count of watchers:14
   - Count of forks:9
 * NPM download stats (optional):
   - Downloads in the last day:4
   - Downloads in the last week:93
   - Downloads in the last month:325

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by bluepnume** [comment]
Just saw this -- let me know if there's anything needed from me! Thanks

**Comment by PeterDaveHello** [comment]
@bluepnume Thanks!

**Comment by kennynaoh** [comment]
@x09326 Sorry for late response. Please rebase to the latest master branch. Thanks!

**Comment by x09326** [comment]
@kennynaoh I have rebased it.

**Comment by PeterDaveHello** [comment]
Does this one meet the popularity requirement?

File: ajax/libs/post-robot/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "krakenjs",
+    "kraken"
+  ],
+  "license": "Apache-2.0",
+  "dependencies": {
+    "babel-eslint": "^6.0.4",
+    "es6-promise": "^3.1.2",
+    "es6-promise-min": "^2.0.1",
+    "sync-browser-mocks": "^1.0.3"
+  },
```
Comment (PeterDaveHello) [inline ajax/libs/post-robot/package.json]: dependencies should be removed

**Comment by x09326** [comment]
@kennynaoh @PeterDaveHello Sorry I forgot to check the popularity, and I have removed the `dependencies`

**Comment by kennynaoh** [comment]
@x09326 Please fix the NPM download stats in comment table.

**Comment by x09326** [comment]
@kennynaoh I have fixed it.

**Comment by kennynaoh** [comment]
@PeterDaveHello Is this worth to add? Or we need to wait for it's popularity. Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh wait for a while

**Comment by kennynaoh** [comment]
@PeterDaveHello Okay

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by dakshshah96** [comment]
We can re-create this PR when the library has enough popularity since this PR has become quite stagnant. Closing the PR and marking issue as "waiting".

**Comment by sufuf3** [comment]
@dakshshah96 please take a look this link: https://www.npmjs.com/package/post-robot
it's popular.
PS. 11,397 downloads in the last month

**Comment by dakshshah96** [comment]
@sufuf3 Sorry about that. Didn't notice that.

**Comment by sufuf3** [comment]
@PeterDaveHello could we close this PR, because #11449 is working on?
Thank you.

**Comment by PeterDaveHello** [comment]
@sufuf3 good idea.

**Pull request closed without merging**